presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/PermissionMetadata.java

```diff
@@ -54,6 +56,6 @@ public boolean isGrantOption()
 
     public HivePrivilegeInfo toHivePrivilegeInfo()
     {
-        return new HivePrivilegeInfo(permission, grantOption);
+        return new HivePrivilegeInfo(permission, grantOption, new PrestoPrincipal(USER, "admin"));
```

Comment:
This class is used in FileHiveMetastore only. And grantor is not supported there. This PR is already huge. Let's implement grantor support for FileHiveMetastore as a separate PR.